test/unit/host_test.rb

```diff
@@ -2058,6 +2058,20 @@ def to_managed!
     end
   end # end of context "location or organizations are not enabled"
 
+  test "should not save if owner taxonomies do not match host configuration" do
+    org1 = FactoryGirl.create(:organization)
+    loc1 = FactoryGirl.create(:location)
```

Comment:
we use `my_organizations` scope that does the query so we need it to exist in DB

Also one generic a bit off-topic comment for reviews, I don't think this is a way to systematically speed-up tests. It already happened that changing `create` to `build` made the test useless. If we want to have unit tests run in seconds we should avoid using ActiveRecord and do either full stubbing (yuck) or have separate layer for storage and business logic (yes please) which is unrealistic IMHO.